angel-data/Web Onboarding NO - Combo.json

```
@@ -0,0 +1,393 @@
+{
+  "id": "5696df77-6e1f-42ff-9585-b3b3ff64ac0c",
+  "ifid": "D3AACABF-A05A-4221-BACB-46035E945B9A",
+  "keywords": {
+    "backButton": "{STORY_KEYWORDS_BACK_BUTTON}",
+    "selectActionSelectLabel": "{STORY_KEYWORDS_SELECT_ACTION_LABEL}",
+    "tooltipModalInformationLabel": "{STORY_KEYWORDS_TOOLTIP_MODAL_INFORMATION_LABEL}"
+  },
+  "lastUpdate": "2022-01-31T12:31:26.248Z",
+  "locales": [
+    "en_NO",
+    "nb_NO
```

oops, missing a closing quotation mark

```suggestion
    "nb_NO"
```